spec/features/visitors/visitor_can_create_an_account_spec.rb

```diff
@@ -26,7 +26,7 @@
     end
 
     it 'notifies me of any info errors, and autofills my form with previous answers' do
-      expect(current_path).to eq(new_user_path)
+      expect(current_path).to eq(register_path)
 
```

Comment:
right now we have duplicate routes going to user#new and user#create. I have just been using the `new_user` path for most of my stuff. We just need to make a decision on which of these two routes to use and get rid of the other in our code base.

Reply:
register